fetch.bs

```diff
@@ -3811,6 +3814,54 @@ the request.
  <var>request</var>'s <a for=request>origin</a> to  <var>request</var>'s <a for=request>client</a>'s
  <a for="environment settings object">origin</a>.
 
+ <li><p>Let <var>hasCustomHeaders</var> be false.
+
+ <li><p><a for=list>For each</a> <var>header</var> in <var>request</var>'s
+ <a for=request>header list</a>, if <var>header</var>'s <a for=header>name</a> is
+ not `<code>If-Modified-Since</code>`, `<code>If-None-Match</code>`, `<code>If-Unmodified-Since</code>`,
+ `<code>If-Match</code>`, `<code>User-Agent</code>`, or `<code>Cache-Control</code>`, set
+ <var>hasCustomHeaders</var> to true.
+
+ <li>
+  <p>If all of the following conditions are true:</p>
+
+  <ul class=brief>
+   <li><p><var>request</var>'s <a for=request>allow preloads</a> is true
+
+   <li><p><var>request</var>'s <a for=request>origin</a> is <var>request</var>'s
+   <a for=request>client</a>'s <a for="environment settings object">origin</a>
+
+   <li><p><var>request</var>'s <a for=request>URL</a>'s <a for=url>scheme</a> is an
+   <a>HTTP(S) scheme</a>
+
+   <li><p><var>request</var>'s <a for=request>mode</a> is "<code>same-origin</code>",
+   "<code>cors</code>", or "<code>no-cors</code>"
+
+   <li><p><var>hasCustomHeaders</var> is false
+
+   <li><p><var>request</var>'s <a for=request>window</a> is not null
+
+   <li><p><var>request</var>'s <a for=request>method</a> is `<code>GET</code>`
+  </ul>
+
+  <p>then perform the following steps:
+
+  <ol>
+   <li><p>Let <var>onPreloadedResponseAvailable</var> be to an algorithm that runs the following
+   step <a>in parallel</a> given <a for=/>response</a> <var>response</var>:
+   Run <a>fetch finale</a> given <var>response</var> and <var>fetchParams</var>.
```

Comment:
Wrapping needs to be adjusted here. Run should be lowercase.

Reply:
OK